### PR TITLE
Edit PyProject Toml PR Message

### PR DESCRIPTION
**Pull request by haohaocreates**
This is what the revised draft looks like, lmk what you guys think!

# Add pyproject.toml for Custom Node Registry

We are working with dr.lt.data and comfyanon to build a global registry for custom nodes (similar to PyPI). Eventually, the registry will be used as a backend for the UI-manager. All nodes go through a verification process before being published to users.

The main benefits are that authors can
- publish nodes by version and users can safely update nodes knowing ahead of time if their workflows will break or not
- automate testing against new commits in the comfy repo and existing workflows through our CI/CD dashboard

Action Required:

- [ ] Go to the [registry](https://registry.comfy.org). Login and create a publisher id. Add the publisher id into the pyproject.toml file.
- [ ] Write a short description.
- [ ] Merge the separate Github Actions PR and run the workflow.

If you want to publish the node manually, [install the cli](https://docs.comfy.org/comfy-cli/getting-started#install-cli) and run `comfy node publish`

Check out our [docs](https://docs.comfy.org/registry/overview#introduction) if you want to know more about the registry. Otherwise, feel free to message me on discord at haohao_81202 or join our [server](https://discord.com/invite/comfyorg) if you have any questions!

### CODE REVIEW

**Review by robinjhuang — COMMENTED** [review]
Let's change all the urls to registry.comfy.org + docs.comfy.org

File: templates/add-toml.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
-- [ ] Go to the [registry](https://comfyregistry.org/). Login and create a publisher id. Comment it here and we will update the PR (or just merge it and change it yourself).
-- [ ] Write a short the description.
+- [ ] Go to the [registry](https://comfyregistry.org/). Login and create a publisher id. Add the publisher id into the pyproject.toml file.
+- [ ] Write a short description.
 - [ ] Merge the separate Github Actions PR and run the workflow.
 
 If you want to publish the node manually, [install the cli](https://comfydocs.org/comfy-cli/getting-started#install-cli) and run `comfy node publish`
-Please message me on [Discord](https://discord.com/invite/comfyorg) if you have any questions!
+
+Check out our [docs](https://comfydocs.org/registry/overview#introduction) page or message me on [Discord](https://discord.com/invite/comfyorg) if you have any questions!
```
Comment (robinjhuang) [inline templates/add-toml.md:17]: Maybe include your discord name?

**Comment by haohaocreates** [comment]
@robinjhuang lmk how this looks 🍌🍌🍌🍌🍌

**Comment by snomiao** [comment]
plz move the old version to ./template/outdate/add-toml-v2.yaml to enable auto-edit

**Review by snomiao — APPROVED** [review]
Nice job

**Pull request merged**